eahub/profiles/views.py

```diff
@@ -80,6 +83,23 @@ def get_type(self):
         return "profile"
 
 
+class SendProfileMessageView(SendMessageView):
+    def form_valid(self, form):
+        recipient = Profile.objects.get(slug=self.kwargs["slug"])
+        message: dict = form.cleaned_data["your_message"]
+        sender_email: dict = form.cleaned_data["your_email_address"]
```

Comment:
are you sure it's those are `dict` btw?

in any case thank you for the annotations - I love that stuff 😁

Reply:
@mattg95  :) ?